storeinterface/storeinterface.go

```diff
@@ -0,0 +1,145 @@
+package storeinterface
```

Comment:
where is this used?

Reply:
It's similar to the store uploader in log-service except without the `makeURL` stuff. We can use it in `store-cli.go` to serialize and upload files to the store I guess?